modules/nf-httpfs/src/main/nextflow/file/http/XFileSystem.groovy

```diff
@@ -44,6 +44,8 @@ class XFileSystem extends FileSystem {
 
     private URI base
 
+    protected XFileSystem(){}
```

Comment:
Let's add a comment why is needed and that it should not generally be used

Reply:
Added in las commit